output.tf

```diff
@@ -1,34 +1,33 @@
 output "name" {
-  value = "${azurerm_template_deployment.namespace.name}"
+  value = azurerm_servicebus_namespace.servicebus_namespace.name
 }
 
 # primary connection string for send and listen operations
 output "primary_send_and_listen_connection_string" {
-  value = "${azurerm_template_deployment.namespace.outputs["primarySendAndListenConnectionString"]}"
+  value = azurerm_servicebus_namespace_authorization_rule.servicebus_authorization_rule.primary_connection_string
 }
 
 # secondary connection string for send and listen operations
 output "secondary_send_and_listen_connection_string" {
-  value = "${azurerm_template_deployment.namespace.outputs["secondarySendAndListenConnectionString"]}"
+  value = azurerm_servicebus_namespace_authorization_rule.servicebus_authorization_rule.secondary_connection_string
 }
 
 # primary shared access key with send and listen rights
 output "primary_send_and_listen_shared_access_key" {
-  value = "${azurerm_template_deployment.namespace.outputs["primarySendAndListenSharedAccessKey"]}"
+  value = azurerm_servicebus_namespace_authorization_rule.servicebus_authorization_rule.primary_key
 }
 
 # secondary shared access key with send and listen rights
 output "secondary_send_and_listen_shared_access_key" {
-  value = "${azurerm_template_deployment.namespace.outputs["secondarySendAndListenSharedAccessKey"]}"
+  value = azurerm_servicebus_namespace_authorization_rule.servicebus_authorization_rule.secondary_key
 }
 
 # sku type
 output "sku" {
-  value = "${var.sku}"
+  value = var.sku
 }
 
-# zoneRedundant
-output "zoneRedundant" {
-  value = "${var.zoneRedundant}"
+# zone redundant
+output "zone_redundant" {
```

Comment:
this is a breaking change (renaming the field), will it affect anyone?

Reply:
This attribute is used in one repo, so reverted the change.